specification/elasticsan/resource-manager/Microsoft.ElasticSan/stable/2023-01-01/elasticsan.json

```diff
@@ -746,12 +770,12 @@
             "in": "body",
             "required": true,
             "schema": {
-              "$ref": "#/definitions/Volume"
+              "$ref": "#/definitions/VolumeCreateParameter"
```

Comment:
All such kinds of changing parameter ref will be breaking changes for SDKS. @mikekistler , I wonder whether they are reviewed by board?

Reply:
> Had this been the only change, I don't think it would be flagged for breaking change review because "ReferenceRedirection" is only reported as a warning and not an error. But the pipeline should detect breaking changes in the SDKs and flag those for review.

Ray has helped to create a request to add it to breaking change tool. Could you help to re-review and approve such breaking changes in this PR?

Reply:
Please send email to @azbreakchangereview with a link to the ADO item.

Reply:
@PreetPRG could you do the breaking change review again?